source/gui/dpiScalingHelper.py

```diff
@@ -21,7 +21,7 @@ def scaleSize(scaleFactor: float, size: _Size) -> _ScaledSize:
 	return int(scaleFactor * size)
 
 
-def getScaleFactor(windowHandle: int) -> float:
+def getScaleFactor(windowHandle: int) -> int:

```

Comment:
do you mind also changing `scaleSize` conversion from `int` to `round`. Using `int` was a mistake on my part in #15523